soc/v2/rockchip/rk3568/Kconfig

```diff
@@ -2,15 +2,12 @@
 # Copyright 2022 openEuler SIG-Zephyr
 # SPDX-License-Identifier: Apache-2.0
 
-choice
-prompt "Rockchip RK3568 SoC"
-depends on SOC_SERIES_RK3568
-
-config SOC_RK3568
-	bool "Rockchip rk3568"
+config SOC_SERIES_RK3568
 	select ARM64
 	select CPU_CORTEX_A55
 	select ARM_ARCH_TIMER
 	select GIC_V3
+	select SOC_FAMILY_ROCKCHIP
```

Comment:
I think SoC family selection can go to `Kconfig.soc`.

Reply:
Fixed